keepsorted/keep_sorted_test.go

```diff
@@ -179,8 +180,8 @@ foo
 			if gotAlreadyFixed != tc.wantAlreadyFixed {
 				t.Errorf("alreadyFixed diff: got %t want %t", gotAlreadyFixed, tc.wantAlreadyFixed)
 			}
-			if len(gotWarnings) != 0 {
-				t.Errorf("Fix returned warnings, expected none:\n%v", gotWarnings)
+			if diff := cmp.Diff(tc.wantWarnings, messages(gotWarnings), cmpopts.EquateEmpty()); diff != "" {
```

Comment:
Nit: r.e., cmpopts.EquateEmpty, what do you think of just making `messages()` use nil slices? Rationale: https://google.github.io/styleguide/go/decisions#nil-slices. Not like the utterly minor performance optimization of allocating the slice size perfectly makes any difference in a test.

Reply:
Done